run_build_impl.sh

```diff
@@ -118,7 +124,12 @@ fi
 echo "-----------------------------"
 
 # Dependencies: Install using rosinstall or list of repositories from the build-job config.
-CATKIN_SIMPLE_URL=git@github.com:catkin/catkin_simple.git
+
+if $CHECKOUT_CATKIN_SIMPLE; then
+  CATKIN_SIMPLE_URL=""
```

Comment:
Added another if guard on the .rosinstall below. :) Some percentage less likely to break everything now.